pkg/pgmodel/ingestor/trace/operation_batch.go

```diff
@@ -77,3 +78,22 @@ func (o operationBatch) GetID(serviceName, spanName, spanKind string) (pgtype.In
 
 	return id, nil
 }
+
+func getPGKindEnum(pk pdata.SpanKind) (string, error) {
+	switch pk {
+	case pdata.SpanKindClient:
+		return "client", nil
+	case pdata.SpanKindServer:
+		return "server", nil
+	case pdata.SpanKindInternal:
+		return "internal", nil
+	case pdata.SpanKindConsumer:
+		return "consumer", nil
+	case pdata.SpanKindProducer:
+		return "producer", nil
+	case pdata.SpanKindUnspecified:
+		return "unspecified", nil
+	default:
+		return "unspecified", fmt.Errorf("unknown span kind: %v", pk)
```

Comment:
Should we return empty value here? I guess that is safer since "unspecified" is a valid value but we are erroring here so if somebody ignores the error, we could potentially have wrong data.